Use nextstrain/ingest

victorlin opened the pull request:
### Description of proposed changes

Begin usage of centralized ingest scripts and add details on how to pull new updates from the central repo.

Note that this repo only uses one of the scripts pulled from the central ingest repo.

### Testing

- [x] Ingest workflow runs locally without errors

victorlin committed:
Describe subrepo setup

The previous commit was created by the following command:

    git subrepo clone https://github.com/nextstrain/ingest ingest/vendored

Add a section in the README on how to use this directory in the future.

victorlin committed:
Use centralized apply-geolocation-rules

Remove the copies in this repo and update references.

The centralized script is functionally identical.

jameshadfield reviewed (approved):
This is a draft but I think it can (and should!) be merged. Thanks!

victorlin commented on ingest/vendored/apply-geolocation-rules:
This PR needs to be updated to use https://github.com/nextstrain/ingest/pull/13.

victorlin committed:
git subrepo clone (merge) https://github.com/nextstrain/ingest ingest/vendored

subrepo:
  subdir:   "ingest/vendored"
  merged:   "5d90818"
upstream:
  origin:   "https://github.com/nextstrain/ingest"
  branch:   "main"
  commit:   "5d90818"
git-subrepo:
  version:  "0.4.6"
  origin:   "https://github.com/ingydotnet/git-subrepo"
  commit:   "110b9eb"
